mesonbuild/environment.py

```diff
@@ -215,6 +215,8 @@ def detect_cpu_family(compilers):
         trial = 'x86'
     elif trial == 'bepc':
         trial = 'x86'
+    elif trial.startswith('arm64'):
```

Comment:
let's just use `==` instead of startswith.

Also, please add a comment that this is for OpenBSD, so that it doesn't get factored out later.